README.md

```diff
@@ -172,4 +172,4 @@ More commonly used terms can be found [here](docs/terminology.md).
 * [hVCF format specifications](docs/hvcf_specifications.md)
 * [haplotype region handling](docs/hvcf_region_handling.md)
 * [PHGv2 terminology](docs/terminology.md)
-* [PHGv2 architecture](docs/img/architecture/phg_v2_architecture-Diagram.drawio.svg)
+  * [PHGv2 architecture](docs/img/architecture/phg_v2_architecture_20240411.svg)
```

Comment:
There's a space there that my OCD doesn't like ;)

Reply:
Fixed - I do not know how this happened. I will blame Intellij...

Reply:
Thanks Brandon!